lenskit/algorithms/als.py

```diff
@@ -126,6 +126,32 @@ def _train_matrix_lu(mat: _CSR, this: np.ndarray, other: np.ndarray, reg: float)
 
     return np.sqrt(frob)
 
+#@njit(parallel=True, nogil=True)
+def _train_bias_row_lu(items, ratings, other, reg):
+    """
+    Args:
+        items(np.ndarray[i64]): the item IDs the user has rated
+        ratings(np.ndarray): the user's (normalized) ratings for those items
+        other(np.ndarray): the item-feature matrix
+        reg(float): the regularization term
+
+    Returns:
+        np.ndarray: the user-feature vector (equivalent to V in the current LU code) 
```

Comment:
Are you using Visual Studio Code? If so, install the EditorConfig extension so it can read the `.editorconfig` file and prevent these kinds of issues in the future.